news/_posts/core-weekly/2022-07-25-coreweekly-29-2022.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 29 of 2022"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2022-07-26
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 18th to Sunday 24th of July 2022.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+The maintainers team was very busy these last few days because of the [major security vulnerability](https://build.prestashop.com/news/major-security-vulnerability-on-prestashop-websites/) discovered last week. If you have not read this article, please read it now.
+
+Following the vulnerability discovery, [PrestaShop version 1.7.8.7 was delivered](https://build.prestashop.com/news/prestashop-1-7-8-7-maintenance-release/) and contains a fix that strengthens MySQL Smarty cache storage against code injection attacks.
+
+While the security work on 1.7.8.7 happened, the 8.0.0 Beta release got much closer to delivery, thanks to everyone from the PrestaShop community! Only a few remaining items are left before we can publish and host a Beta testing period.
```

Comment:
Perhaps link to https://devdocs.prestashop.com/1.7/project/release/minor-release-lifecycle/#public-beta-period or copy-paste a subset

Reply:
Done in https://github.com/PrestaShop/prestashop.github.io/pull/1104/commits/e10189a85dabd5b88d344619827df2be263cc03c